### DISCUSSION TITLE
Help css

### DISCUSSION THREAD

**Discussion by JenniClark** [community/community#113312] [Other Feature Feedback, Questions, & Ideas]
tips in css?

**Comment by pablitoSeija** [comment]
Hi..  Always use relative units and check what's the default display type of tag you are using. Making display flex and then playing elements in the web browser in Inspect elements can help you decide what you should choose.

Instead of using fixed pixel values for layout widths, use percentages. This allows elements to adjust their size relative to the screen width.

.container {
width: 100%;
max-width: 1200px; /* Optional: set a maximum width for larger screens /
margin: 0 auto; / Centers the container */
}

Media queries are CSS statements that allow you to apply different styles based on specific screen size conditions. You can target devices based on width, height, orientation, and resolution.

That's all
